buildpacks/nodejs/bin/build

```diff
@@ -20,6 +20,11 @@ info "Node.js Buildpack"
 
 rm -rf node_modules
 
+install_or_reuse_toolbox "$layers_dir/toolbox"
```

Comment:
toolbox needs to be installed first so that yj can be used to parse store.toml file